### ISSUE TITLE
Fix layout of Espanol button

### ISSUE THREAD

**Issue by kostajh** [durhamatletico/durhamatletico-cms#133]
Follow up to #123 

![screenshot from 2016-06-22 15-42-44](https://cloud.githubusercontent.com/assets/208893/16280997/3d18381c-3890-11e6-8768-d033376d08d6.png)

Probably makes sense for it to be inline...

@AnneTee if you have time could you help with this?

**Comment by AnneTee** [comment]
@kostajh Yes definitely!

**Comment by AnneTee** [comment]
@kostajh This is a pretty quick fix but I'm currently not able to see changes to `styles.css` on my local site. For reference...

- I hadn't worked on the site in a while so I completely re-cloned it, added my local settings file, and imported a fresh DB dump and files from production
- I'm using compass and my SCSS changes are being correctly compiled into `styles.css`
- I've tried rebuilding the cache, and I've got the render cache turned off via local settings

In fact, I can't get any changes to the theme (e.g. template updates) to show in the browser. It obviously seems like a cache thing, but I'm pretty sure I've got them all turned off...any ideas?

One thing I noticed: thought I have the following lines in `settings.local.php`

```
/**
 * Disable CSS and JS aggregation.
 */
$config['system.performance']['css']['preprocess'] = FALSE;
$config['system.performance']['js']['preprocess'] = FALSE;
```

...it appears CSS and JS aggregation are turned on and the checkboxes are disabled:
<img width="1181" alt="screen shot 2016-06-22 at 9 06 32 pm" src="https://cloud.githubusercontent.com/assets/8465998/16288756/4d440806-38bd-11e6-84e9-b46dcbb9274b.png">

The options on the public files directory page they suggest don't seem useful either.

Quite possible it's something small I missed, but I'm out of ideas for the moment!
